pkg/storaged/jobs-panel.jsx

```diff
@@ -145,11 +145,25 @@ export class JobsPanel extends React.Component {
             return false;
         }
 
-        var jobs = Object.keys(client.jobs).filter(job_is_stable).sort(cmp_job);
+        var jobs = [ ];
+        var have_reminder = false;
+        for (var p in client.jobs) {
+            console.log(p);
```

Comment:
Probably don't want to leave this in, but i'll remove when merging.

Reply:
Yes, thanks!

Reply:
I have removed that line.